x11-packages/xfce4-cpugraph-plugin/build.sh

```diff
@@ -0,0 +1,14 @@
+TERMUX_PKG_HOMEPAGE=https://docs.xfce.org/panel-plugins/xfce4-cpugraph-plugin/start
+TERMUX_PKG_DESCRIPTION="Graphical representation of the CPU load"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="1.2.10"
+TERMUX_PKG_REVISION=1
```

Comment:
No revision if the package is just being added.

Reply:
There are two commits (with the first one to keep the `top` implememtation in commit history, and the second one to revise to use `/proc/#/task/#/stat`). Subject to your practice, there may be no need to add revision. Thx.

Reply:
There are two commits (with the first one to keep the `top` implememtation in commit history, and the second one to revise to use `/proc/#/task/#/stat`). Subject to your practice, there may be no need to add revision. Thx.